pkg/apiserver/handlers.go

```diff
@@ -392,10 +392,16 @@ func WithAuthorizationCheck(handler http.Handler, getAttribs RequestAttributeGet
 	})
 }
 
-// APIRequestInfo holds information parsed from the http.Request
-type APIRequestInfo struct {
-	// Verb is the kube verb associated with the request, not the http verb.  This includes things like list and watch.
-	Verb       string
+// RequestInfo holds information parsed from the http.Request
+type RequestInfo struct {
+	// IsResourceRequest indicates whether or not the request is for an API resource
```

Comment:
> What value does it have for sub-resource requests?

Resource in the context or authorization has meant, "accessing an api resource that follows kube norms like prefix, version, resource, name, etc"